### ISSUE TITLE
Deploy the web scraper to a production environment. 

### ISSUE THREAD

**Issue by knox-academy** [knox-academy/webscraping#39]
Mike McConnelly: Great addition, Dan! Another issue title we could use is "Configure production environment for web scraper deployment".

Issue Description - Before we can deploy the web scraper to a production environment, we need to configure the environment to ensure that it is compatible with the application. This includes setting up any necessary databases, servers, and other infrastructure components required for the web scraper to function properly.

How to Test - We can test the completion of this issue by verifying that the production environment has been properly configured and that the web scraper is able to function as expected. This can be done through manual testing or automated testing tools.
